src/js/api/brand.js

```diff
@@ -147,7 +147,19 @@ ripe.Ripe.prototype.get3dSceneEnvironmentUrl = function(options) {
  * @returns {String} The URL of the specified initials texture.
  */
 ripe.Ripe.prototype.getTextureMapUrl = function(map, options) {
-    return null;
+    options = this._getInitials3dOptions(options);
+
+    const texture = {
+        pattern: options.baseTexture,
+        displacement: options.displacementTexture,
+        metallic: options.metallicTexture,
+        normal: options.normalTexture,
+        roughness: options.roughnessTexture
+    }[map];

```

Comment:
Would it make sense to do

```javascript
const textureMaps = {
   ...
}
const texture = textureMaps[map];
```

Reply:
I did this this way to avoid using switch 🤷‍♂️ 

But if you prefer that I do it this way, is good for me!

Reply:
Without switch, the same but just having the object in one variable and the result of lookup in another.